pyproject.toml

```
@@ -1,5 +1,42 @@
 # SPDX-License-Identifier: MIT
 
+[build-system]
+requires = ["setuptools>61"]
+build-backend = "setuptools.build_meta"
+
+[project]
+name = "disnake"
+description = "A Python wrapper for the Discord API"
+readme = "README.md"
+authors = [
+  {name = "Disnake Development" }
+]
+requires-python = ">=3.8"
+keywords = ["disnake", "discord", "discord api"]
+license = { file = "LICENSE" }
+classifiers = [
+    "Development Status :: 5 - Production/Stable",
+    "License :: OSI Approved :: MIT License",
+    "Intended Audience :: Developers",
+    "Natural Language :: English",
+    "Operating System :: OS Independent",
+    "Programming Language :: Python :: 3.8",
+    "Programming Language :: Python :: 3.9",
+    "Programming Language :: Python :: 3.10",
+    "Topic :: Internet",
+    "Topic :: Software Development :: Libraries",
+    "Topic :: Software Development :: Libraries :: Python Modules",
+    "Topic :: Utilities",
+    "Typing :: Typed",
+]
+dynamic = ["version", "dependencies", "optional-dependencies"]
+
+[project.urls]
+Homepage = "https://github.com/DisnakeDev/disnake"
+Documentation = "https://docs.disnake.dev/"
+GitHub ="https://github.com/DisnakeDev/disnake"
+"Issue tracker" = "https://github.com/DisnakeDev/disnake/issues"
```

```suggestion
Repository = "https://github.com/DisnakeDev/disnake"
Documentation = "https://docs.disnake.dev/"
Changelog = "https://docs.disnake.dev/page/whats_new.html"
"Issue tracker" = "https://github.com/DisnakeDev/disnake/issues"
```
Might as well update these while we're at it.

---

I think that keeping some of the ones that were suggested to remove might be good to keep. I may also add a few more. 

This is the list of urls that get a special icon, if you're wondering. https://github.com/pypi/warehouse/blob/main/warehouse/templates/packaging/detail.html

---

https://github.com/DisnakeDev/disnake/pull/830/commits/02ac1951f283387b08b3687f70870c6490b3a3e2

---

> This is the list of urls that get a special icon

Yup, that's the file I was looking at as well.

I don't really see a reason to have both "Homepage" and "Repository" pointing to the same site, which is why I had removed one of them in the suggestion above.

---

The odd thing is I ordered it Changelog/Homepage/Documentation/Repository

---

In terms of [spec](https://github.com/toml-lang/toml/blob/main/toml.md#table), toml tables are technically unordered, I'm guessing tomli just happens to preserve the order. They're ordered correctly in the sdist/wheel metadata, so that's not it.

Looks like pypi just sorts them alphabetically? The only control you have over the ordering is that the `Home-Page`/`Download-URL` core metadata fields are always on top:
https://github.com/pypi/warehouse/blob/2116945657693114231d22697d7ec7f6cfe08578/warehouse/packaging/models.py#L555-L556
Those two aren't part of PEP 621, only the `Project-URL` field can be set through that :/

According to https://github.com/pypi/warehouse/issues/3097 it was once just the reverse order, but that doesn't appear to be the case anymore.

---

Hm, in that case it might make sense to remove homepage altogether. I wanted to keep it because I thought it would stay at the top, but that's only the case for packages that use the home-page: field.

---

https://github.com/DisnakeDev/disnake/pull/830/commits/ab74ade1734bddc1645ea2f773b2e620c11a3165